examples/translations/pt-br-all.toml

```diff
@@ -110,10 +110,10 @@ other = "longitude inválida"
 # Notifications messages.
 
 [notification_replied]
-other = "⚠️ *Notificação*\n%s respondeu uma mensagem sua em *%s*\n\n_%s_"
+other = "⚠️ *Notificação*\n[%s](tg://user?id=%d) respondeu uma mensagem sua em *%s*\n\n_%s_"
```

Comment:
Portuguese question (not sure): Respondeu *a* uma mensagem sua or Respondeu uma mensagem sua?

Reply:
Used "a uma" in this case. Seem better.